app/main.py

```diff
@@ -1,6 +1,43 @@
+import os
+
+import requests
+from requests import RequestException
+
+
+URL = "http://api.weatherapi.com/v1/current.json"
+API_KEY = os.environ["API_KEY"]
```

Comment:
It's important to check if the `API_KEY` environment variable is set before using it. Consider adding a check to handle cases where it might not be set to avoid potential `KeyError`.